### PR TITLE
Document enable_implicit_transaction_for_batch_statements setting

### PR DESCRIPTION
**Pull request by rafiss**
fixes https://github.com/cockroachdb/docs/issues/13254
fixes https://github.com/cockroachdb/docs/issues/13258
fixes https://github.com/cockroachdb/docs/issues/13269

### CODE REVIEW

File: v22.2/transactions.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ transaction to retry, the client needs to be involved in retrying the whole
 transaction and so you should write your transactions to use
 [client-side intervention](#client-side-intervention).
 
+**New in v22.2:** The [`enable_implicit_transaction_for_batch_statements` session variable](#set-vars) defaults to `true`. This means that any batch of statements is treated as an implicit transaction, so the `BEGIN`/`COMMIT` commands are not needed to group all the statements in one transaction.
```
Comment (rmloveland) [inline v22.2/transactions.md:131]: I think to get the build green this link to `#set-vars` needs to be updated to point to `set-vars.html#enable-implicit-transaction-for-batch-statements`

**Review by rmloveland — APPROVED** [review]
LGTM modulo the build failure (commented below with what I think will fix it)

**Comment by rmloveland** [comment]
@rafiss thank you for this PR BTW!!!1!

**Comment by rafiss** [comment]
thanks for the review! just wanted to fix this since it came up in a support ticket question

**Pull request merged**